Automattic/wp-calypso
# Use LocaleProvider in calypso

#### Changes proposed in this Pull Request

* Implements usage of LocaleProvider, useLocalizeUrl
* Implements withLocalizeUrl 
* Makes use of both with/useLocalizeUrl
* adds i18n-utils to domain-picker package

#### Testing instructions

* Check url is localized as shown in #45610
* Check urls is localized on the privacy settings screen
* Check existing localized urls still localize elsewhere in calypso
 
Fixes #45610 :tada:
